charts/stable/shinobi/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+appVersion: 1.0.0
```

Comment:
This needs to be equal to the image tag. In this case I guess that would be `dev`.